tests/unit/GoFields/GoFieldSubmit.spec.js

```diff
@@ -1,25 +1,87 @@
 import { createLocalVue, shallowMount } from '@vue/test-utils'
 import * as AgoUikit from '../../../src'
 import { GoFieldSubmit } from '../../../src'
-import renderCases from './GoFieldSubmit.render-cases'
 
 const localVue = createLocalVue()
 
 localVue.use(AgoUikit)
 
 describe('GoFieldSubmit', () => {
+  let wrapper
+  let buttonSubmit
+
+  beforeEach(() => {
+    wrapper = shallowMount(GoFieldSubmit, {
+      localVue,
+    })
+
+    buttonSubmit = wrapper.find({ref: 'submit'})
+  })
+
   // Install
   it('should be installed', () => {
     expect(localVue.options.components['GoFieldSubmit']).toBeTruthy()
   })
 
-  // Render testing
-  // renderCases.forEach(renderCase => {
-  //   const [fromComp, toHtml] = renderCase
+  it('should be default options component', () => {
+    expect(buttonSubmit.is('button')).toBe(true)
+
+    expect(buttonSubmit.attributes('type')).toBe('submit')
+  })
+
+  it('should have empty attributes which no fill', () => {
```

Comment:
should have empties attributes which no fill

Reply:
should have empty attributes which are not filled